.travis.yml

```diff
@@ -0,0 +1,21 @@
+sudo: required
+
+language: java
+
+services:
+  - docker
+
+before_install:
+  - wget http://mirrors.ae-online.de/apache/maven/maven-3/3.3.9/binaries/apache-maven-3.3.9-bin.tar.gz
+  - tar xf apache-maven-3.3.9-bin.tar.gz
+  - export M2_HOME=$PWD/apache-maven-3.3.9
+  - export PATH=$M2_HOME/bin:$PATH
```

Comment:
Maven version is not 3.2. It's 3.1.1.

![screenshot from 2017-01-10 13-46-34](https://cloud.githubusercontent.com/assets/7941617/21798605/637cca00-d73b-11e6-8cfb-c894cb997173.png)